apps/portal/js/gadget.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+$(function () {
+    /**
+     * Find a given component in the current page
+     * @param {Number} id
+     * @returns {Object}
+     * @private
+     */
+    var findComponent = function (id) {
+        var i;
+        var length;
+        var area;
+        var component;
+        var components;
+
+        var content = page.content.default;
+        for (area in content) {
+            if (content.hasOwnProperty(area)) {
+                components = content[area];
+                length = components.length;
+                for (i = 0; i < length; i++) {
+                    component = components[i];
+                    if (component.id === id) {
+                        return component;
+                    }
+                }
+            }
+        }
+    };
+
+    /**
+     * To render the specific gadget in a certain page
+     */
+    var renderGadget = function () {
+        window.onresize = function () {
+            location.reload();
+        };
+        var com = $('.emb-gadget');
+        var id = window.location.pathname.split("/").pop();
+        console.log(ues.global.dbType);
```

Comment:
Thanks for pointing out. This was already removed with 0844673ddf4fb482a4234277c1b7da3ef2036eea